File: python/Lesson0007/Lesson0007.py
# Mở tệp để đọc
file = open("test.txt", "r")

# Đọc nội dung tệp
noidung = file.read()
print("Nội dung tệp:")
print(noidung)

# Đóng tệp
file.close()

# Mở tệp để ghi
file = open("test.txt", "w")

# Ghi nội dung vào tệp
file.write("Xin chào, thế giới!")

# Đóng tệp
file.close()

# Mở tệp để đọc và ghi
file = open("test.txt", "r+")

# Đọc nội dung tệp
noidung = file.read()
print("Nội dung tệp sau khi ghi:")
print(noidung)

# Ghi thêm nội dung vào tệp
file.write(" Đây là nội dung được thêm vào.")

# Đóng tệp
file.close()

# Xóa tệp
import os
os.remove("test.txt")

# Tạo thư mục
import os
os.mkdir("thumuc")

# Xóa thư mục
import os
os.rmdir("thumuc")

File: Cpp/Lesson0007/Lesson0007.cpp
#include <iostream>
#include <fstream>
#include <string>

using namespace std;

int main() {
    // Tạo một tệp tin mới để ghi
    ofstream tepTinGhi("example.txt");
    if (tepTinGhi.is_open()) {
        // Ghi nội dung vào tệp tin
        tepTinGhi << "Xin chào, thế giới!";
        tepTinGhi.close();
        cout << "Ghi tệp tin thành công!" << endl;
    } else {
        cout << "Không thể mở tệp tin để ghi!" << endl;
    }

    // Đọc nội dung từ tệp tin
    ifstream tepTinDoc("example.txt");
    if (tepTinDoc.is_open()) {
        string dong;
        while (getline(tepTinDoc, dong)) {
            // In nội dung đọc được
            cout << "Nội dung tệp tin: " << dong << endl;
        }
        tepTinDoc.close();
    } else {
        cout << "Không thể mở tệp tin để đọc!" << endl;
    }

    return 0;
}